app/cmd/root.go

```diff
@@ -62,7 +58,11 @@ func init() {
 }
 
 func initConfig() {
-	if rootOptions.ConfigFile == "" {
+	fmt.Println("Jenkins CLI (jcli) manage your Jenkins")
```

Comment:
This will cause every single command to print it. This welcome sentence is expected in the root cmd.